app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute, which is fine, but ensure that it is used correctly to store references to `Person` objects by their names.